lib/zhr_devs/identity_management/identity.ex

```diff
@@ -26,6 +26,18 @@ defmodule ZhrDevs.Identity do
     GenServer.start_link(__MODULE__, success_struct, name: via_tuple(hashed_identity))
   end
 
+  def renew_login(hashed_identity) when is_binary(hashed_identity) do
+    GenServer.call(via_tuple(hashed_identity), :renew_login)
+  end
+
+  @spec parse_hashed_identity(String.t()) :: Uptight.Base.Urlsafe.t()
```

Comment:
Don't like String.t here. It normally means that you expect printable string, but why base-encode it then? 

If you go the opposite way: take an encoded value and want to decode it into a raw binary and store the result in the Uptight.Base.UrlsafeSixtyFour, then you probably should rename this function to "digest_to_urlsafe". I would also add an Uptight.Text function head which would extract String.t out of the Text.t object.

This way, to any reader it'd be clear that when you say String.t, you really mean it.